src/Backend/OnboardingInsuranceAPI/Areas/User/UserData.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace OnboardingInsuranceAPI.Areas.User
+{
```

Comment:
Make sure you are using .NET 6 C# 10 file scoped namespaces.

Reply:
Oh, my fault.